indico/MaKaC/webinterface/tpls/ConferenceCreation.tpl

```diff
@@ -7,10 +7,20 @@
 
     <div class="groupTitle">${ _("Step 1: Choose a category")}</div>
 
-    <div style="padding: 10px">
-        <input type="hidden" value="${ categ['id'] }" name="categId" id="createCategId"/>
-        <span class="selectedCategoryName">${ _("The event will be created in:")} <span id="categTitle" class="categTitleChosen">${ categ['title'] }</span></span><input ${'style="display: none;"' if nocategs else ""} id="buttonCategChooser" type="button" value="${ _("Browse...")}" onclick="openCategoryChooser()"/>
-    </div>
+    <table class="groupTable" id="event-creation-category-field">
+        <tr>
+            <td nowrap class="titleCellTD">
+                <span class="titleCellFormat">${ _("Category")}</span>
+            </td>
+            <td>
+                <div class="form-group">
+                    <div class="form-field">
+                        ${ template_hook('event-category-field') }
```

Comment:
Why `event-creation-...`? we have `event-location-field`, `event-person-links-field`, `event-references-list`, so keeping the same scheme it should be `event-category-field`